app/models/vm_or_template.rb

```diff
@@ -231,18 +231,34 @@ def from_infra_manager?
   REQUIRED_ADVANCED_SETTINGS = {
-    'vmi.present'         => [:paravirtualization,   :boolean],
-    'vmsafe.enable'       => [:vmsafe_enable,        :boolean],
-    'vmsafe.agentAddress' => [:vmsafe_agent_address, :string],
-    'vmsafe.agentPort'    => [:vmsafe_agent_port,    :integer],
-    'vmsafe.failOpen'     => [:vmsafe_fail_open,     :boolean],
-    'vmsafe.immutableVM'  => [:vmsafe_immutable_vm,  :boolean],
-    'vmsafe.timeoutMS'    => [:vmsafe_timeout_ms,    :integer],
-    'entitled_processors' => [:entitled_processors,  :float],
-    'processor_type'      => [:processor_share_type, :string],
-    'pin_policy'          => [:processor_pin_policy, :string],
-    'placement_group'     => [:placement_group,      :string],
-    'software_licenses'   => [:software_licenses,    :string],
+    'vmi.present'                => [:paravirtualization,         :boolean],
+    'vmsafe.enable'              => [:vmsafe_enable,              :boolean],
+    'vmsafe.agentAddress'        => [:vmsafe_agent_address,       :string],
+    'vmsafe.agentPort'           => [:vmsafe_agent_port,          :integer],
+    'vmsafe.failOpen'            => [:vmsafe_fail_open,           :boolean],
+    'vmsafe.immutableVM'         => [:vmsafe_immutable_vm,        :boolean],
+    'vmsafe.timeoutMS'           => [:vmsafe_timeout_ms,          :integer],
+    'entitled_processors'        => [:entitled_processors,        :float],
+    'processor_type'             => [:processor_share_type,       :string],
+    'pin_policy'                 => [:processor_pin_policy,       :string],
+    'placement_group'            => [:placement_group,            :string],
+    'software_licenses'          => [:software_licenses,          :string],
+    'ppt_ratio'                  => [:ppt_ratio,                  :string],
+    'enforce_affinity_check'     => [:enforce_affinity_check,     :boolean],
+    'proc_units'                 => [:proc_units,                 :float],
+    'dedicated_proc'             => [:dedicated_proc,             :boolean],
+    'dedicated_sharing_mode'     => [:dedicated_sharing_mode,     :string],
+    'uncapped'                   => [:uncapped,                   :boolean],
+    'processor_compatibility'    => [:processor_compatibility,    :boolean],
+    'shared_weight'              => [:shared_weight,              :integer],
+    'availability_priority'      => [:availability_priority,      :integer],
+    'boot_volume_type'           => [:boot_volume_type,           :string],
+    'storage_connectivity_group' => [:storage_connectivity_group, :string],
+    'ame_expansion_factor'       => [:ame_expansion_factor,       :string],
+    'secure_boot'                => [:secure_boot,                :integer],
+    'srr_capability'             => [:srr_capability,             :boolean],
+    'shared_proc_pool_name'      => [:shared_proc_pool_name,      :string],
+    'enable_lpar_metric'         => [:enable_lpar_metric,         :boolean],
```

Comment:
This all looks unrelated, accidentally pushed?

Reply:
seems that to be the case. Not sure why these are showing in here....

Reply:
Looks like you based your branch off of another one that had this commit in it https://github.com/ManageIQ/manageiq/pull/21750/commits/239f6ddf9a558547ccc45bf118584a4cdbc657ff